### ISSUE TITLE
in Cardboard Mode not displaying well in mobile

### ISSUE THREAD

**Issue by smitshah81** [pchen66/panolens.js#32]
I guess there is issue with height and width in cardboard (Mobile).I have also tested in mobile full screen mode.

**Comment by BrunoFenzl** [comment]
I can confirm this one. Tested on mobile Android chrome 51-52 and Safari iOS 9.3.2 and 9.3.3. the handler onWindowResize in viewer.js get called multiple times always increasing the canvas sizes.
I tried to stop this behaviour setting the a css rule with max-width/height to 100% and it stopped the growth but the canvas width/height attributes are still wrong.

Here is a screenshot that shows that the inline styles get updated, but not the attributes:
<img width="831" alt="screen shot 2016-08-09 at 22 26 08" src="https://cloud.githubusercontent.com/assets/1745180/17532378/9482b2e2-5e80-11e6-9744-49da636a3774.png">

Any quick fix or hint for that? Any help more than appreciated!

**Comment by smitshah81** [comment]
hmm.. i have checked in Motorola G2 and Samsung S6 and I got issue in both Video as well as audio.
but if it is related to window resize then it should be fixed [this](https://github.com/pchen66/panolens.js/issues/21#issuecomment-228971067) way "I guess...." 
by calling this after window resize

**Comment by BrunoFenzl** [comment]
Thanks for the link. I will give it a try later and post the results

**Comment by BrunoFenzl** [comment]
Hi! This code is already in place. What got my problem solved was using the canvas.clientWidth/Height and settings its size to 100% in css as seen [here](https://github.com/mrdoob/three.js/issues/4903).

```
        html, body,
    	.panolens-container{
    		width: 100%;
    		height: 100%;
    		overflow: hidden; /* optional */
    		margin: 0; /* remove body margins */
    	}

    	canvas{
    		width: 100%;
    		height: 100%;
    	}
``` 

And in viwer.js around line 946


```
PANOLENS.Viewer.prototype.onWindowResize = function () {

		var width, height, expand;

		**width = this.renderer.domElement.clientWidth;
		height = this.renderer.domElement.clientHeight;**

		this.camera.aspect = width / height;
		this.camera.updateProjectionMatrix();
		this.renderer.setSize( width, height );
```

It's definitely a solution for all cases, but since I'm using it on a Iframe, it's good for me.

**Comment by pchen66** [comment]
Closed #82 
Please reopen if you experience the issue again
